app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  def options_for_video_reviews(selected = nil)
+    options = [5,4,3,2,1].map { |number| [pluralize(number, "Star")]}
```

Comment:
Also `[pluralize(number, "Star")]` is not 100% good solution because you are going to pass `"X Stars"` to your controller from `select`.
This is how your HTML will look approx:

```html
<select ... >
  <option value="5 Stars">5 Stars</option>
  <option value="4 Stars">4 Stars</option>
  <option value="3 Stars">3 Stars</option>
</select>
```

the second element in the array should be pure integer rating - without "Stars", so it will be used as `"value"` - thats what your `select` tag will pass to `form` and that should be submitted to the controller.

The benefit of this approach is that when your database has already some rating saved, it will retrieve it and use it as pre-selected. What will be useful in your `queue_items#index` view.